client/api/index.js

```diff
@@ -24,3 +24,11 @@ export function getUnconfirmed (cb) {
     cb(null, data.body)
   })
 }
+
+export function newBooking (data, cb) {
+  request.post('api/v1/user/addbooking')
```

Comment:
A new booking api call must go through the login function as it needs to attach the authorization token.  This function would not be needed.